api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -124,6 +124,18 @@ message Bootstrap {
     LogFormat log_format = 1;
   }
 
+  message DeferredStatOptions {
+    // When the flag is enabled, Envoy will lazily initialize a subset of the stats (see below).
+    // This will save memory and CPU cycles when creating the objects that own these stats, if those
+    // stats are never referenced throughout the lifetime of the process. However, it will incur additional
+    // memory overhead for these objects, and a small increase of CPU usage when a at least one of the stats
+    // is updated for the first time.
+    // Groups of stats that will be lazily initialized:
+    //  * Cluster traffic stats: a subgroup of the :ref:`cluster statistics <config_cluster_manager_cluster_stats>`
```

Comment:
Just a sanity check: this PR does change the cluster-traffic stats to be deferred, right?

Reply:
Not this PR; that will be in the next PR. Maybe we should leave this as a "pending PR #xxxx" comment and adjust that line in that PR?

Reply:
Add not-implemented-hide tag, as noted in the comment above. This can be removed later